### PR TITLE
HEEDLS-482 Remove clickable link from admin card when user doesn't ha…

### PR DESCRIPTION
**Pull request by AlexJacksonDS**
…ve permissions

Added a new claim helper method for the CentreManager permission (same permissions as the Administrators page)

Used this to disable the link in the Admins card on the dashboard page.

Refactored each section of the Dashboard view into a partial view.

image

### CODE REVIEW

**Review by davidm-m — APPROVED** [review]
Nice use of partials, looks good!

**Comment by stellake** [comment]
Can you post an image of this version to the hee external channel to see if HEE is happy with the design?

File: DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Dashboard/Index.cshtml [hunk 1]
```diff
@@ -22,209 +22,10 @@
     <h1 class="nhsuk-heading-xl" id="app-page-heading">Centre dashboard</h1>
     <p class="nhsuk-body-m">Welcome <span class="nhsuk-u-font-weight-bold">@Model.FirstName</span>. You are managing <span class="nhsuk-u-font-weight-bold">@Model.CourseCategory</span> courses.</p>
 
-  <ul class="nhsuk-grid-row nhsuk-card-group dashboard-4-1-card-group">
+    <partial name="_Dashboard4x1CardGroup" model="Model"/>
```
Comment (stellake) [inline DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Dashboard/Index.cshtml:25]: Nice! I like the partials split but I wonder if we could rename it into something more intuitive - I would probably find it easier to find the partial I want to modify without having to go to the Index page, if we had DashboardTopCardGroup/BottomCardGroup partials? Then, if the style of these section changes to say 3x2 we wouldn't have to rename these either 🤔

**Review by stellake — APPROVED** [review]
Looks good - a small comment about naming + running this past HEE! 👍

**Comment by AlexJacksonDS** [comment]
Confirmation from Kevin in slack that it is OK.

**Pull request merged**